python/llm/src/ipex_llm/transformers/models/mistral.py

```diff
@@ -730,7 +737,14 @@ def mistral_attention_forward_4_36_quantized(
                     "please make sure to initialize the attention class "
                     "with a layer index."
                 )
-            kv_seq_len += past_key_value.get_usable_length(kv_seq_len, self.layer_idx)
+            # [SnapKV] add kv_seq_len
+            if hasattr(self, "kv_seq_len"):
+                if self.kv_seq_len != 0:
+                    kv_seq_len += self.kv_seq_len
+                else:
+                    kv_seq_len += past_key_value.get_usable_length(kv_seq_len, self.layer_idx)
+            else:
```

Comment:
Do we still need this?

Reply:
The quantized kv version is WIP, I will update it later.